src/features/dashboard/pages/Dashboard.tsx

```diff
@@ -1,3 +1,211 @@
+      },
+      portfolioShare: 12,
+    },
+    {
+      holding: {
+        holdingIcon: 'https://logo.clearbit.com/google.com',
+        holdingSymbol: 'GOOGL',
+        holdingName: 'Alphabet Inc.',
+      },
+      stocksPurchased: 8,
+      purchaseValue: {
+        total: 2000,
+        perShare: 250,
+      },
+      currentValue: {
+        total: 2200,
+        perShare: 275,
+      },
+      profit: {
+        absolute: 200,
+        percentage: 10,
+      },
+      portfolioShare: 15,
+    },
+    {
+      holding: {
+        holdingIcon: 'https://logo.clearbit.com/amazon.com',
+        holdingSymbol: 'AMZN',
+        holdingName: 'Amazon.com, Inc.',
+      },
+      stocksPurchased: 5,
+      purchaseValue: {
+        total: 1650,
+        perShare: 330,
+      },
+      currentValue: {
+        total: 1500,
+        perShare: 300,
+      },
+      profit: {
+        absolute: -150,
+        percentage: -9.09,
+      },
+      portfolioShare: 8,
+    },
+    {
+      holding: {
+        holdingIcon: 'https://logo.clearbit.com/tesla.com',
+        holdingSymbol: 'TSLA',
+        holdingName: 'Tesla, Inc.',
+      },
+      stocksPurchased: 20,
+      purchaseValue: {
+        total: 1200,
+        perShare: 60,
+      },
+      currentValue: {
+        total: 1400,
+        perShare: 70,
+      },
+      profit: {
+        absolute: 200,
+        percentage: 16.67,
+      },
+      portfolioShare: 20,
+    },
+    {
+      holding: {
+        holdingIcon: 'https://logo.clearbit.com/nvidia.com',
+        holdingSymbol: 'NVDA',
+        holdingName: 'NVIDIA Corporation',
+      },
+      stocksPurchased: 12,
+      purchaseValue: {
+        total: 600,
+        perShare: 50,
+      },
+      currentValue: {
+        total: 720,
+        perShare: 60,
+      },
+      profit: {
+        absolute: 120,
+        percentage: 20,
+      },
+      portfolioShare: 10,
+    },
+    {
+      holding: {
+        holdingIcon: 'https://logo.clearbit.com/coca-cola.com',
+        holdingSymbol: 'KO',
+        holdingName: 'The Coca-Cola Company',
+      },
+      stocksPurchased: 30,
+      purchaseValue: {
+        total: 450,
+        perShare: 15,
+      },
+      currentValue: {
+        total: 480,
+        perShare: 16,
+      },
+      profit: {
+        absolute: 30,
+        percentage: 6.67,
+      },
+      portfolioShare: 5,
+    },
+    {
+      holding: {
+        holdingIcon: 'https://logo.clearbit.com/jnj.com',
+        holdingSymbol: 'JNJ',
+        holdingName: 'Johnson & Johnson',
+      },
+      stocksPurchased: 20,
+      purchaseValue: {
+        total: 1800,
+        perShare: 90,
+      },
+      currentValue: {
+        total: 1900,
+        perShare: 95,
+      },
+      profit: {
+        absolute: 100,
+        percentage: 5.56,
+      },
+      portfolioShare: 12,
+    },
+  ];
+
+  return (
+    <section className="container mx-auto px-4 pb-12">
+      <header className="flex flex-row justify-between py-6">
```

Comment:
This is in reality not a `header`. But your current header should receive this attribute as that now is a `div`